pageserver/src/tenant/upload_queue.rs

```diff
@@ -189,6 +192,7 @@ impl UploadQueue {
             dangling_files: HashMap::new(),
             shutting_down: false,
             shutdown_ready: Arc::new(tokio::sync::Semaphore::new(0)),
+            last_aux_file_policy: RuntimeAuxFilePolicy::Unspecified,
```

Comment:
This would certainly be better with default. Note I've put `latest_lineage` clumped up there instead of growing the list downwards.

I hope to replace this "flattened IndexPart fields" by maintaining a `dirty_index_part: IndexPart` in future instead, which is no longer cloned etc. is only serialized once.

Reply:
So you did not move it?

Reply:
https://github.com/neondatabase/neon/pull/7668/commits/9dfb8b9ff45d0a12ea5bba91180173b809e23a1f it was `None` and is now `Default::default`. I did not move it to `dirty_index_part` because it looks like a larger change orthogonal to this PR.